odl/solvers/advanced/proximal_operators.py

```diff
@@ -568,6 +588,33 @@ def proximal_cconj_l2(space, lam=1, g=None):
 
     Notes
     -----
+    The functional :math:`F` is given by
+
+        .. math::
+
+            F(x) = \\lambda ||x - g||_2
+
+    The convex conjugate :math:`F^*` of :math:`F` is given by
+
+        .. math::
+
+            F^*(y) = \\left\{ \\begin{array}{ll}
+            0 & \\quad  ||y-g||_2 \leq \\lambda, \\\\
+            \\infty & \\text{else.}
+            \\end{array} \\right.
+
+    For a step size :math:`\\sigma`, the proximal operator of
+    :math:`sigma F^*` is given by the projection onto the set
+    :math:`||y-g||_2 \leq \\lambda`, i.e., by
+
+        .. math::
+
+            prox[\\sigma F^*](y) = \\left\{ \\begin{array}{ll}
+            \\lambda \\frac{y - g}{||y - g||}
```

Comment:
`lambda` has an effect, since that decides the region for the indicator function. But `sigma` do not mattes, since (positive) scaling the indicator function is still the indicator function.